src/mgmt/cli_mgmt.c

```diff
@@ -403,6 +403,8 @@ int dc_mgmt_net_cfg(const char *name)
 		/* If the client env didn't provide a domain, we can assume we don't need one. */
 		if (ofi_domain)
 			D_INFO("Using client provided OFI_DOMAIN: %s\n", ofi_domain);
+		else
+			D_WARN("Client didn't provide OFI_DOMAIN, assuming we don't need one");
```

Comment:
I'm not sure about logging this message at WARN level. This is not an error condition unless they are using a provider that requires a domain (like verbs). You could print something at INFO level, for example, "Using client-provided OFI_INTERFACE without OFI_DOMAIN," but I'm not sure how useful it is, given all the env variables are printed at INFO level anyway.

Reply:
if client does not provide ofi_domain then cart automatically assumes  ofi_domain == ofi_interface. this works for all providers, except for verbs and cxi, where domains and interface names do not match. this is why those providers require additional setting. 

It probably would be better to warn in the cart layer at init time, if (provider == cxi or verbs_rxm) and ofi_domain is not specified then print the warning there

Reply:
Makes sense.